snappy/install.go

```diff
@@ -46,6 +46,8 @@ const (
 	AllowGadget
 	// DeveloperMode will install the snap without confinement
 	DeveloperMode
+	// TryMode indicates the snap is in try (unpacked directory) mode
+	TryMode
```

Comment:
@niemeyer  yes, now for migrating we need to still use/take care of the high values they got here (I think we don't need any of the others actually, maybe the GC one). wondering if storing bit fields in the state is a good idea in general though, maybe flags should be stored as some mapping or some simple string repr, seems more future proof